libraries/config/src/config.cpp

```diff
@@ -144,6 +146,14 @@ FullNodeConfig::FullNodeConfig(Json::Value const &string_or_object, Json::Value
     network.network_boot_nodes.push_back(node);
   }
 
+  light_node_history = getConfigDataAsUInt(root, {"light_node_history"}, true, 0);
+
+  // Min history for light node is 110% of kDagExpiryPeriodLimit to avoid any race condition around the
+  // kDagExpiryPeriodLimit
+  if (light_node_history > 0 && light_node_history < kDagExpiryPeriodLimit * 1.1) {
+    light_node_history = kDagExpiryPeriodLimit * 1.1;
+  }
```

Comment:
I think it's better to make kDagExpiryPeriodLimit to 1100.
Then:
if (light_node_history != 0) {
  light_node_history = std::min(light_node_history, kDagExpiryPeriodLimit);
}

Reply:
this part of code changed, please review again